test/test_image.py

```diff
@@ -419,6 +420,7 @@ def test_encode_jpeg_errors():
         encode_jpeg(torch.empty((100, 100), dtype=torch.uint8))
 
 
+@pytest.skipif(IS_MACOS, reason="https://github.com/pytorch/vision/issues/8031")
```

Comment:
In theory we would only skip for M1 and not all darwin, but it doesn't really matter since the non-M1 builds will be dropped very soon.